Directory.Build.targets

```diff
@@ -6,6 +6,14 @@
     <SupportedPlatform Remove="iOS" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!--
+      The System.Threading.Tasks global import interferes with Microsoft.Build.Utilities.Task that is used extensively
+      in this repository. Remove it to avoid the conflict.
+    -->
+    <Import Remove="System.Threading.Tasks" />
```

Comment:
@dsplaisted / @wli3 this was fairly annoying to deal with running tests using the dogfood build of the SDK. Thoughts?